### PR TITLE
feat:added installation page

### PR DESCRIPTION
**Pull request by stutisk**
image

### CODE REVIEW

**Review by Hamdrive — APPROVED** [review]
Good use of inline comments, structure of code is also well done.
Ready to merge!

File: components/installation/installation.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            </div>
+            <!-- content -->
+            <div class="content">
+                <div class="content-title">
+                    Installation
+                </div>
+
+                <div class="content-subtitle">
+
+                </div>
```
Comment (Tanay-J) [inline components/installation/installation.html]: I think you missed removing this empty ```div```

**Review by Tanay-J — APPROVED** [review]
Very readable code, did not find any issues. You are ready to merge.

**Review by shuklahimanshu81 — APPROVED** [review]
Everything looks good. Good to merge.

**Pull request merged**